django_stubs_ext/django_stubs_ext/patch.py

```diff
@@ -46,15 +46,24 @@ def __repr__(self) -> str:
 ]
 
 
-# currently just adds the __class_getitem__ dunder. if more monkeypatching is needed, add it here
 def monkeypatch() -> None:
     """Monkey patch django as necessary to work properly with mypy."""
+
+    # Add the __class_getitem__ dunder.
     suited_for_this_version = filter(
         lambda spec: spec.version is None or VERSION[:2] <= spec.version,
         _need_generic,
     )
     for el in suited_for_this_version:
         el.cls.__class_getitem__ = classmethod(lambda cls, *args, **kwargs: cls)
 
+    # Define reveal_type/reveal_locals, to not cause NameError during setting
+    # up Django.
+    if not TYPE_CHECKING:
```

Comment:
We assume that all code under `monkeypatch` is called during runtime and not type-checking.